localstack/services/cloudformation/cloudformation_starter.py

```diff
@@ -763,6 +737,19 @@ def Kinesis_Stream_get_cfn_attribute(self, attribute_name):
 
     kinesis_models.Stream.get_cfn_attribute = Kinesis_Stream_get_cfn_attribute
 
+    # patch cloudformation backend describe_change_set(..)
+    # #760 cloudformation deploy invalid xml error
+    cloudformation_backend_describe_change_set_orig = CloudFormationBackend.describe_change_set
+
+    def cloudformation_backend_describe_change_set(self, change_set_name, stack_name=None):
+        change_set = cloudformation_backend_describe_change_set_orig(self, change_set_name, stack_name)
+        if change_set.status == 'REVIEW_IN_PROGRESS':
+            change_set.status = 'CREATE_COMPLETE'
```

Comment:
Trying to understand this change - shouldn't the stack status only change once the change set is executed via `execute_change_set`? 

> If you create a change set for a new stack, AWS Cloudformation creates a stack with a unique stack ID, but no template or resources. The stack will be in the REVIEW_IN_PROGRESS state until you execute the change set. 

https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateChangeSet.html